### PR TITLE
Tests for CASSANDRA-7814

### PR DESCRIPTION
**Pull request by stef1927**
Some tests I wrote quite some time ago for CASSANDRA-7814, but never merged.

### CODE REVIEW

File: cqlsh_tests/cqlsh_tests.py [hunk 1]
```diff
@@ -506,6 +506,179 @@ def test_list_queries(self):
 (6 rows)
 """)
 
+    @require("7814")
+    @since('2.1')
+    def test_describe(self):
+        self.cluster.populate(1)
```
Comment (ptnapoleon) [inline cqlsh_tests/cqlsh_tests.py]: Can you add a doxygen docstring that includes `@jira_ticket CASSANDRA-7814`?

File: cqlsh_tests/cqlsh_tests.py [hunk 2]
```diff
@@ -506,6 +506,179 @@ def test_list_queries(self):
...
+        self.cluster.start()
+
+        node1, = self.cluster.nodelist()
+        node1.watch_log_for('thrift clients...')
```
Comment (ptnapoleon) [inline cqlsh_tests/cqlsh_tests.py]: Remove this line in favor of the modification to line 513.

File: cqlsh_tests/cqlsh_tests.py [hunk 3]
```diff
@@ -506,6 +506,179 @@ def test_list_queries(self):
 (6 rows)
 """)
 
+    @require("7814")
+    @since('2.1')
+    def test_describe(self):
+        self.cluster.populate(1)
+        self.cluster.start()
```
Comment (ptnapoleon) [inline cqlsh_tests/cqlsh_tests.py]: Pass the argument `wait_for_binary_proto=True` to start

**Comment by stef1927** [comment]
Thanks for the comments Philip, I pushed a new commit.

**Pull request merged**